scripts/zones/Arrapago_Reef/mobs/Bukki.lua

```diff
@@ -2,11 +2,38 @@
 -- Area: Arrapago Reef
 --   NM: Bukki
 -----------------------------------
-mixins = { require('scripts/mixins/families/imp') }
+mixins =
+{
+    require('scripts/mixins/job_special'),
+    require('scripts/mixins/families/imp'),
+}
 -----------------------------------
 ---@type TMobEntity
 local entity = {}
 
+entity.onMobSpawn = function(mob)
+    xi.mix.jobSpecial.config(mob, {
+        specials =
+        {
+            { id = xi.jsa.MANAFONT, hpp = math.random(10, 50) },
+        },
+    })
+
+    mob:setTP(3000)
+    mob:setMobSkillAttack(0)
+    mob:setMagicCastingEnabled(true)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 300)
+    mob:setMobMod(xi.mobMod.GIL_MAX, -1)
+    mob:setLocalVar('tp_spam', math.random(5, 15))
+    mob:addListener('COMBAT_TICK', 'BUKKI_TICK', function(mob)
+        if mob:getHPP() <= mob:getLocalVar('tp_spam') and mob:getLocalVar('tp_spam') > 0 then
+            mob:setLocalVar('tp_spam', 0)
+            mob:setMagicCastingEnabled(false)
+            mob:setMobSkillAttack(788)
+        end
+    end)
```

Comment:
To fix the CI:
```lua
mob:addListener('COMBAT_TICK', 'BUKKI_TICK', function(mobArg)
        if mobArg:getHPP() <= mobArg:getLocalVar('tp_spam') and mobArg:getLocalVar('tp_spam') > 0 then
            mobArg:setLocalVar('tp_spam', 0)
            mobArg:setMagicCastingEnabled(false)
            mobArg:setMobSkillAttack(788)
        end
    end)
```

Reply:
Athough, you could always just use the regular entity function for combat tick, i would think

Reply:
ah thanks, couldnt figure out what it wanted, we dont have to use args